solutions/fully-configurable/variables.tf

```diff
@@ -94,6 +94,16 @@ variable "vpn_name" {
   nullable    = false
 }
 
+variable "client_auth_methods" {
+  type        = string
+  description = "The methods used to authenticate VPN clients to this VPN server. Allowable values are: certificate, username. For more information, see https://cloud.ibm.com/docs/vpc?topic=vpc-vpn-client-environment-setup"
```

Comment:
The link doesn't seems right, I think for authentication this one is more appropiate - https://cloud.ibm.com/docs/vpc?topic=vpc-client-to-site-authentication

Reply:
not exactly, I am sharing a link, where consumer can see how to use username/password or certificated authentication. That description also helped to understand where to add cert using open VPN. 
Your link is more about how to create certificate.